packages/firebase_ml_vision/lib/src/vision_cloud_detector_options.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_ml_vision;
+
+/// Options for cloud vision detectors.
+///
+class VisionCloudDetectorOptions {
+  /// Constructor for [VisionCloudDetectorOptions].
+  ///
+  const VisionCloudDetectorOptions(
+      {this.maxResults = 10, this.modelType = MODEL_TYPE_STABLE})
+      : assert(maxResults > 0),
+        assert(
+            modelType == MODEL_TYPE_LATEST || modelType == MODEL_TYPE_STABLE);
+
+  static const int MODEL_TYPE_STABLE = 1;
```

Comment:
In dart we typically prefer `lowerCamelCase` for constants.

Reply:
Fixed in 4969a6dff